lib/sheerid_api.rb

```diff
@@ -0,0 +1,65 @@
+# Service object for interacting with SheerID's API
+# docs found at: http://developer.sheerid.com/rest-api
+class SheeridAPI
+  DEFAULT_WEBHOOK_URL = 'https://dev.openstax.org/accounts/i/sheerid/webhook'
+  AUTHORIZATION_HEADER = "Bearer #{Rails.application.secrets.sheerid_api_secret}"
+  HEADERS = {
+    'Authorization': AUTHORIZATION_HEADER,
+    'Accept': 'application/json',
+    'Content-Type': 'application/json'
+  }.freeze
+
+  PROGRAM_ID = ENV['SHEERID_PROGRAM_ID']
+  BASE_URL = 'https://services.sheerid.com/rest/v2'
+
+  WEBHOOKS_URL = "#{BASE_URL}/program/#{PROGRAM_ID}/webhook"
+  INFO_URL = "#{BASE_URL}/info"
+  PROGRAM_THEME_URL = "#{BASE_URL}/#{PROGRAM_ID}/theme"
+
+  class << self
+    def get_verification_details(verification_id)
```

Comment:
imo, for a longer list of methods, it's better to do the def self.x way of class methods instead of this.  a long train of these makes is harder to follow

Reply:
I'm not sure I understand 🤔